src/pages/scientificServices/pipelines/utils/input-utils.ts

```diff
@@ -7,15 +7,18 @@ export interface PipelineInputDescription {
   label?: string;
   placeholder?: string;
   helpText?: string;
+  validationRegex?: string;
```

Comment:
this is another property that we'd probably want to serve up from the backend when we add the rest of these properties to the db. the one thing to keep in mind is that there are some minor differences in regex syntax between java and javascript

Reply:
theres a good chance we wont be doing much validation outside of type (like float, int, etc) and file suffixes.

Reply:
gotcha, that makes sense!